packages/bygger/src/translations/utils.ts

```diff
@@ -89,6 +89,9 @@ const extractTextsFromProperties = (props: NavFormType['properties']): TextObjec
   return array;
 };
 
+const getValues = (values?: Array<{ label: string; value: string; description?: string }>) =>
```

Comment:
getLabelsAndDescriptions() for å beskrive hva som returneres?

Og burde vi filtrere bort eventuelle duplikater her (i praksis kan man vel ha lik description her), eller gjøres det allerede et annet sted?

Reply:
Endra til getLabelsAndDescriptions() 👍 
De filtreres bort et annet sted, siden beskrivelsen kan være brukt i en annen komponent. Testa at det ikke dukker opp duplikater.